### PR TITLE
Fix for connectionlog

### PR DESCRIPTION
**Pull request by karle-nishi**
## Summary

1. Fix for using same ConnectionLog class throughout
2. Fix for test cases

### CODE REVIEW

File: core/extract/extract_parser.py [hunk 1]
```diff
@@ -35,33 +35,36 @@ def parse_log(
...
+            log_file, connections, last_connections, start_time, end_time
+        )
     elif "start_node" in filename:
         logger.debug(f"Parsing start node log: {filename}")
         _parse_start_node_log(log_file, logs, databases, start_time, end_time)
 
 
 def _parse_user_activity_log(file, logs, databases, start_time, end_time):
-    user_activity_log = Log()
+    user_activity_log = StartNodeLog()
```
[the remark this answers is not in the document]
Comment (sathiish-kumar) [reply core/extract/extract_parser.py:47]: Maybe this is the reason why it was named `Log` to begin with?

[the remark this answers is not in the document]
Comment (karle-nishi) [reply core/extract/extract_parser.py:47]: yes correct. Will update the changes

**Review by DevanathanSabapathy1 — APPROVED** [review]

File: core/extract/extract_parser.py [hunk 2]
```diff
@@ -30,12 +30,14 @@ def parse_log(
     :param start_time: start_time of extract
     :param end_time: end_time of extract
     """
-    if "useractivitylog" in filename:
+    if "Log" in filename:
```
Comment (sathiish-kumar) [inline core/extract/extract_parser.py:33]: I don't think this is intentional?

File: core/tests/test_extractor.py [hunk 3]
```diff
@@ -205,7 +205,7 @@ def test_save_logs_s3(
             }
         )
         e.save_logs(
-            {"useractivitylog": [self.get_query()]},
+            {"Log": [self.get_query()]},
```
Comment (sathiish-kumar) [inline core/tests/test_extractor.py:208]: This is not intentional right?

File: core/tests/test_extractor.py [hunk 4]
```diff
@@ -234,7 +234,7 @@ def test_save_logs_non_s3(self, mock_copy_replacements, mock_cluster_dict):
                 }
             )
             e.save_logs(
-                {"useractivitylog": [self.get_query()]},
+                {"Log": [self.get_query()]},
```
Comment (sathiish-kumar) [inline core/tests/test_extractor.py:237]: This is not intentional right?

**Review by sathiish-kumar — APPROVED** [review]

**Pull request merged**